server/src/controller/oauth2.ts

```diff
@@ -1,11 +1,10 @@
 import {Context} from "koa";
 import * as googleApis from "googleapis";
-import * as jwt from "jsonwebtoken";
 
 import * as keys from "../../keys/keys.json";
 import {User} from "../entity/user";
-import {getManager, Repository} from "typeorm";
-import { LoginTicket, TokenPayload } from "google-auth-library";
+import { LoginTicket } from "google-auth-library";
+import { UserController } from "./user";
```

Comment:
_Technically_ bad practice, but since we don't have a service layer to handle business logic I'll ignore it :P

Reply:
yeah fair